content/ko/docs/concepts/workloads/controllers/replicationcontroller.md

```diff
@@ -0,0 +1,291 @@
+---
+
+
+
+title: 레플리케이션 컨트롤러
+feature:
+  title: 자가 치유
+  anchor: 레플리케이션 컨트롤러의 동작 방식
+  description: >
+    노드가 죽었을 때 오류가 발생한 컨테이너를 교체하고 재 스케쥴하고, 사용자 정의 상태 체크에 응답하지 않는 컨테이너를 제거하며, 서비스를 제공할 준비가 될 때까지 클라이언트에 이를 알리지 않는다.
+
+content_template: templates/concept
+weight: 20
+---
+
+{{% capture overview %}}
+
+{{< note >}}
+[`레플리카셋`](/docs/concepts/workloads/controllers/replicaset/) 을 구성하는 [`디플로이먼트`](/docs/concepts/workloads/controllers/deployment/) 가 현재 권장되는 레플리케이션 설정방법이다.
```

Comment:
Hi @marcushyungseoklee 

I wonder if it's more useful to leave the Kubernetes resources untranslated:
```suggestion
[`ReplicaSet`](/docs/concepts/workloads/controllers/replicaset/) 을 구성하는 [`Deployment`](/docs/concepts/workloads/controllers/deployment/) 가 현재 권장되는 레플리케이션 설정방법이다.
```

(possibly with Korean translations as hints)